### PR TITLE
hotfix: changed opc-base version to 1.0.0-beta

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Explain the feature/fix

1. changed opc-base version to 1.0.0-beta

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by gisumit — APPROVED** [review]
LGTM

File: packages/opc-base/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@one-platform/opc-base",
-  "version": "1.0-beta",
+  "version": "1.0.0-beta",
```
Comment (hybridx) [inline packages/opc-base/package.json:3]: @akhilmhdh after changing the version you should run `npm i` once again so that even the package-lock is updated

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/opc-base/package.json (hunk 1 above)
Comment (akhilmhdh) [reply packages/opc-base/package.json:3]: Done

Comment (hybridx) [reply packages/opc-base/package.json:3]: @akhilmhdh I don't see the package-lock.json change in these changes

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/opc-base/package.json (hunk 1 above)
Comment (akhilmhdh) [reply packages/opc-base/package.json:3]: Oopz forgot the push command 😄

**Review by hybridx — APPROVED** [review]
LGTM

**Comment by hybridx** [comment]
@akhilmhdh can you create another PR to remove `  "private": true,`

**Comment by akhilmhdh** [comment]
@hybridx  Dang. Ok will put that in a moment

**Pull request merged**